npm/vue/src/index.ts

```diff
@@ -376,7 +376,22 @@ export function mount<
   >['vm']
 }>
 
-// implementation
+/**
+ * Mounts a component and returns an object containing the component and VueWrapper
+ * @param componentOptions
+ * @param options
+ * @returns {Cypress.Chainable<VueWrapper>}
```

Comment:
The return value has been changed in the `release/11.0.0` branch.  It how returns a chainable with two keys: `wrapper` and `component`.  Not sure if this is the best way to document this...
```suggestion
 * @returns {Cypress.Chainable<{wrapper: VueWrapper<T>, component: T}
```